### PR TITLE
LEARNER-4692 Test - iOS - Automate new Landing Screen

### PR DESCRIPTION
**Pull request by nedo**
Following scenarios are covered,
* Verify that Landing screen is loaded successfully
* Verify that Landing screen has following contents,
** edX logo
** Description text-field
** Search Courses edit-field
** Register button
** Sign In button
* Verify that all elements have their default values
* Verify that tapping "Log In" loads Log In screen
* Verify that tapping back icon from "Log In" screen navigate user back to Landing screen
* Verify that tapping "Create Your Account" loads Register screen
* Verify that tapping back icon from "Register" screen navigate user back to Landing screen
* Verify that tapping search after entering <course name> should load "Discovery" screen
* Verify that Discovery screen is showing
* Verify tapping back icon from 'Discovery' screen navigate user back to Landing screen

### CODE REVIEW

File: ios/pages/ios_new_landing.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+        return self.discovery_cancel_button
+
+    def cancel_dicovery_screen(self):
+        """
+        Close/Cancel Dicovery Screen
+        """
+
+        self.discovery_cancel_button.click()
+
```
Comment (raeeschachar) [inline ios/pages/ios_new_landing.py]: I am not certain but would we need a `wait` after this button click?

File: ios/tests/test_ios_new_landing.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (raeeschachar) [inline ios/tests/test_ios_new_landing.py]: Empty line not required

Comment (nedo) [reply ios/tests/test_ios_new_landing.py]: done

**Comment by nedo** [comment]
@staubina can you please review this PR ?

**Pull request merged**